2-mandatory/8-password-validator.js

```diff
@@ -23,7 +23,17 @@ PasswordValidationResult=  [false, false, false, false, true]
 
 */
 
-function validatePasswords(passwords) {}
+function validatePasswords(passwords) {
+  return passwords.map((val, index) => {
+    if (val.length >= 5 &&
+    containsUppercaseLetter(val) && containsLowercaseLetter(val) && containsNumber(val) && containsSymbol(val) && passwords.indexOf(val) === index) {
+      return true
+    } return false;
+  })
+
+
+  
+}
 
```

Comment:
Well implemented password validation.
Please remember to format your code as it will make your code to look nicer and be more readable.